README.md

```diff
@@ -108,6 +108,18 @@ script:
   # etc.
 ```
 
+It might happen that clippy is not available for a certain nightly release.
+In this case you can try to conditionally install clippy from the git repo.
+
+```yaml
+language: rust
+rust:
+  - nightly
+before_script:
+   - rustup component add clippy-preview --toolchain=nightly || cargo install --git https://github.com/rust-lang-nursery/rust-clippy/ --force clippy
```

Comment:
In case when `cargo install` fails too it might be useful to have a boolean like

```sh
if $(install-clippy); then
    HAS_CLIPPY=true
else
    echo 'Could not install clippy'
    HAS_CLIPPY=false
fi
``` 

and for testing use

```sh
if "${HAS_CLIPPY}" cargo clippy
```

Reply:
Hmm, so you don't want the job to fail if clippy is not available?

Reply:
The GitHub link needs to be fixed, then I think this is good to go.

Reply:
updated.